varcode/read_evidence/pileup_collection.py

```diff
@@ -585,3 +591,22 @@ def from_bam(pysam_samfile, loci):
             if close_on_completion:
                 pysam_samfile.close()
 
+def to_locus(variant_or_locus):
+    """
+    Return a Locus object for a Variant instance.
+
+    Since the read evidence module uses a different Variant class than the rest
+    of varcode, this is necessary. This should be removed once varcode switches
+    to interbase coordinates.
+    """
+    if isinstance(variant_or_locus, Locus):
+        return variant_or_locus
+    try:
+        return variant_or_locus.locus
+    except AttributeError:
+        # IMPORTANT: if varcode someday changes from inclusive to interbase
```

Comment:
Nit: maybe mark this with `FIXME` or `TODO` for easy discovery of FIXMEs/TODOs (I should probably put this suggestion in the wiki)